popSimManu.tex

```diff
@@ -270,23 +280,26 @@ \section*{Results}
 deCODE~\citep{kong2010fine} genetic maps for humans;
 the \cite{salome2011recombination} map for \emph{A.~thaliana};
 and the \cite{comeron2012many} map for \emph{D.~melanogaster}.
-Adding futher maps to the library is trivial.
+Adding further maps to the library is straightforward.
 The second key element of a species description
 within \stdpopsim is a set of carefully curated population genetic model
 descriptions from the literature, which allow simulation under
 specific historical scenarios that have been fit to present-day patterns of
-genetic variation. (See the Methods for a description of the community
+genetic variation. Our aim is that this catalog will grow as we continue
+to refine our understanding of different species' demographic histories
+and new models are published. (See the Methods for a description of the community
```

Comment:
this was already said in slightly different words [above](https://github.com/popsim-consortium/manuscript/pull/171/files#diff-d92f49319853667480a7cacf9e3500d2R219)

Reply:
yea...wasnt sure how how pedantic I should be in trying to drive this point home in response to the reviewer's comment

Reply:
no need to say the same thing twice